### ISSUE TITLE
Add NIP05 verification to satdress accont

### ISSUE THREAD

**Issue by cryptoteun** [nbd-wtf/satdress#44]
Satdress is a really nice option to deliver lightning addresses to the mass. 

But wouldn't it be nice if it's also possible to get the same NIP05 nostr verification at for the same address?

**Comment by fiatjaf** [comment]
I don't think that is a good idea, but if someone sends a clean PR I'll merge it.

**Comment by tobiasbaur** [comment]
How about NIP57? Any plans or anybody working on that yet?

**Comment by believethehype** [comment]
I added NIP05 and NIP57 (amongst some other things) to this fork. Happy to send a merge request for whatever of this is wanted. Also need some help to add code to check for paid invoices on backends other than lnd and lnbits.

https://github.com/believethehype/satdress

**Comment by hazrulnizam** [comment]
> I added NIP05 and NIP57 (amongst some other things) to this fork. Happy to send a merge request for whatever of this is wanted. Also need some help to add code to check for paid invoices on backends other than lnd and lnbits.
> 
@believethehype I tried your fork. It worked up until finished registering the first user. After that it gives http panic when GETting `/.well-known/lnurlp/user`.

Can't open an issue on the fork github so I thought I'd let you know here.

**Comment by believethehype** [comment]
Hey, I enabled issues at the fork.  Can you write over there how and with what options enabled you are running it?

**Comment by hazrulnizam** [comment]
> Hey, I enabled issues at the fork. Can you write over there how and with what options enabled you are running it?

https://github.com/believethehype/satdress/issues/2
